frontends/p4/fromv1.0/v1model.h

```diff
@@ -298,6 +306,20 @@ class V1Model : public ::Model::Model {
     DirectMeter_Model   directMeter;
     DirectCounter_Model directCounter;
 
+    static void convertExternObjects(Util::JsonArray *o, BMV2::Backend *bmv2,
```

Comment:
These methods do not belong to this class, they are specific to the BMv2 simulator.
V1model is about a specific architecture, which could be compiled to run on many other targets.